### PR TITLE
fix unicode for ddog

### PR DESCRIPTION
**Pull request by emord**
```
Traceback (most recent call last):
  File "/home/cchq/www/production/current/manage.py", line 118, in <module>
    execute_from_command_line(sys.argv)
  File "/home/cchq/www/production/current/python_env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/cchq/www/production/current/python_env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/cchq/www/production/current/python_env/local/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/cchq/www/production/current/python_env/local/lib/python2.7/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/management/commands/run_ptop.py", line 80, in handle
    start_pillow(pillow)
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/run_pillowtop.py", line 40, in start_pillow
    pillow_instance.run()
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/pillow/interface.py", line 85, in run
    self.process_changes(since=self.get_last_checkpoint_sequence(), forever=True)
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/pillow/interface.py", line 108, in process_changes
    self._record_change_success_in_datadog(change)
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/pillow/interface.py", line 184, in _record_change_success_in_datadog
    self.__record_change_metric_in_datadog('commcare.change_feed.changes.success', change)
  File "/home/cchq/www/production/current/corehq/ex-submodules/pillowtop/pillow/interface.py", line 194, in __record_change_metric_in_datadog
    'domain:{}'.format(change.metadata.domain),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xda' in position 12: ordinal not in range(128)
```

@snopoke @dannyroberts @gcapalbo

### CODE REVIEW

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/ota/views.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def search(request, domain):
             to="{}@{}.com".format('frener', 'dimagi'),
             notify_admins=False, send_to_ops=False
         )
-        _soft_assert(False, "Someone in domain: {} tried accessing case search without a config".format(domain), e)
+        _soft_assert(False, u"Someone in domain: {} tried accessing case search without a config".format(domain), e)
```
Comment (dimagimon) [inline corehq/apps/ota/views.py]: E501 line too long (116 > 115 characters)

**Comment by snopoke** [comment]
@emord thanks for this, added some more places where this might happen. Also fixed here: https://github.com/dimagi/commcare-hq/pull/15314/commits/a6b8abb28e4af941f8afaecf7671c7dd933e4242

File: corehq/apps/indicators/models.py [hunk 2]
```diff
@@ -48,8 +48,8 @@ def __init__(self, _d=None, **kwargs):
         self.class_path = self._class_path
 
     def __str__(self):
-        return "\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
-            " version: %(version)s, namespace: %(namespace)s. ID: %(indicator_id)s." % {
+        return u"\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
```
Comment (emord) [inline corehq/apps/indicators/models.py]: I don't htink this needs the \ at the end

**Review by emord — COMMENTED** [review]
Thanks @snopoke

File: corehq/apps/indicators/models.py (hunk 2 above)
Comment (gcapalbo) [inline corehq/apps/indicators/models.py]: i think this would still need the \, it wouldn't need it only if the whole expression was wrapped in parentheses

**Review by gcapalbo — APPROVED** [review]

File: corehq/apps/indicators/models.py (hunk 2 above)
Comment (emord) [inline corehq/apps/indicators/models.py]: Ah. you're right. I missed the parens https://github.com/dimagi/commcare-hq/pull/15328/files#diff-21bd1c0e52720275d9cf19e3dd556434R1039

**Pull request merged**